utils/src/funTest/kotlin/storage/HttpFileStorageFunTest.kt

```diff
@@ -43,7 +43,7 @@ import org.ossreviewtoolkit.utils.log
 class HttpFileStorageFunTest : WordSpec() {
     private val loopback = InetAddress.getLoopbackAddress()
     private val port = Random.nextInt(1024, 49152) // See https://en.wikipedia.org/wiki/Registered_port.
-        .also { log.debug { "Using port $it for test server." } }
+        .also { log.debug { "Using port $it for HTTP server." } }
```

Comment:
Maybe it would make sense to rewrite this test using WireMock. WireMock directly supports starting a test server at a free random port.

Reply:
I'll leave that as a future exercise then 😁

Speaking about WireMock, since we're using OkHttp I was recently wondering whether we should also prefer to use [MockWebServer](https://github.com/square/okhttp/tree/master/mockwebserver) instead. Here is [some](https://www.baeldung.com/spring-mocking-webclient) [tutorials](https://rieckpil.de/test-spring-webclient-with-mockwebserver-from-okhttp/). Any opinion about that?

Reply:
I had a short look at MockWebServer, and it indeed looks promising. My first impression is that the framework is pretty light-weight and can be easily used. WireMock currently offers more functionality, e.g. a DSL to stub responses for specific requests, matchers on request properties, JSON support, ...

Reply:
... and pre-recording of real responses, right? While *MockWebServer* also has a *RecordedRequest* class, it seems to be something else.